cross-dom-bridge-erc20/index.js

```diff
@@ -12,18 +12,18 @@ const l1Url = `https://eth-goerli.g.alchemy.com/v2/${process.env.GOERLI_ALCHEMY_
 const l2Url = `https://opt-goerli.g.alchemy.com/v2/${process.env.OPTIMISM_GOERLI_ALCHEMY_KEY}`
 
 
-// Contract addresses for DAI tokens, taken
-// from https://static.optimism.io/optimism.tokenlist.json
-const daiAddrs = {
-  l1Addr: "0x4f96fe3b7a6cf9725f59d353f723c1bdb64ca6aa",
-  l2Addr: "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1"
-}    // daiAddrs
+// Contract addresses for OPTb tokens, taken
+// from https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/master/data/OUTb/data.json
+const erc20Addrs = {
+  l1Addr: "0x32B3b2281717dA83463414af4E8CfB1970E56287",
+  l2Addr: "0x3e7eF8f50246f725885102E8238CBba33F276747"
+}    // erc20Addrs
 
 
 
 // Global variable because we need them almost everywhere
 let crossChainMessenger
-let l1ERC20, l2ERC20    // DAI contracts to show ERC-20 transfers
+let l1ERC20, l2ERC20    // OUTb contracts to show ERC-20 transfers
 let addr    // Our address
```

Comment:
Nit: this variable name/comment is not as clear as it could be. 
`let deployer; // this is our L1 address we will deploy from`